File: GLLara/GLLRenderWindow.m
//
//  GLLRenderWindow.m
//  GLLara
//
//  Created by Torsten Kammer on 01.03.13.
//  Copyright (c) 2013 Torsten Kammer. All rights reserved.
//

#import "GLLRenderWindow.h"

#import "GLLView.h"

@implementation GLLRenderWindow

- (GLLCamera *)camera
{
	return self.renderView.camera;
}

@end

File: GLLara/GLLRenderWindow.swift
//
//  GLLRenderWindow.swift
//  GLLara
//
//  Created by Torsten Kammer on 10.07.22.
//  Copyright © 2022 Torsten Kammer. All rights reserved.
//

import Cocoa

@objc class GLLRenderWindow: NSWindow {
    
    @IBOutlet weak var renderView: GLLView?
    
    override func becomeKey() {
        super.becomeKey()
        
        renderView?.windowBecameKey()
    }
    
    override func resignKey() {
        super.resignKey()
        
        renderView?.windowResignedKey()
    }
    
}
